example/get-media-crmls.rb

```diff
@@ -0,0 +1,38 @@
+require 'rets'
+
+client = Rets::Client.new({
+  login_url: 'url',
+  username: 'username',
+  password: 'password',
+  version: 'RETS/1.7.2' 
+})
+
+begin
+    client.login
+rescue => e
+    puts 'Error: ' + e.message
+    exit!
+end
+
+puts 'We connected! Lets get all the photos for a property...'
+
+
+photos = client.find (:all), {
```

Comment:
the parens on `(:all)` aren't needed and could be confusing to people reading the documentation. Mind removing them?